### DISCUSSION TITLE
Description keyword does not validate properly

### DISCUSSION THREAD

**Discussion by jrmcmah2** [python-jsonschema/jsonschema#840] [Q&A] [answered]
So I have the following situation. where I am trying to validate a pretty simple schema: 

`instrument_schema = {
  "cusip": {
    "type": "string"
  },
  "symbol": {
    "type": "string"
  },
  "description": {
    "type": "string"
  },
  "exchange": {
    "type": "string"
  },
  "assetType": {
    "type": "string",
    "enum": [
      "EQUITY",
      "ETF",
      "FOREX",
      "FUTURE",
      "FUTURE_OPTION",
      "INDEX",
      "INDICATOR",
      "MUTUAL_FUND",
      "OPTION",
      "UNKNOWN"
    ]
  }
}`

The payload that I am looking to validate looks like this: 

`test_payload = {
  "AAPL": {
    "cusip": "037833100",
    "symbol": "AAPL",
    "description": "Apple Inc. - Common Stock",
    "exchange": "NASDAQ",
    "assetType": "EQUITY"
  }
}`

I am validating just like this: 

`jsonschema.validate(instance=test_payload, schema=instrument_schema)`

When I run this, I get the following error: 
![image](https://user-images.githubusercontent.com/40073416/134459751-197d1a27-7d4a-4e78-8e31-8a7c5cabbae5.png)

The description in the test payload is clearly a string valie just like exchange or cusip. The interesting part of this is when I change the key in both the payload and schema from "description" to "descriptio" the validation works just fine. This is very weird. I would not expect a particular key value to be a problem, especially one like "description" that is so common in various json payloads. Could anyone provide guidance as to what I may be doing wrong here? 

Additionally if this is not the correct place for this post I do apologize. Please let me know and I will repost in the proper spot.

**Comment by Julian** [comment, marked as answer]
You mean `{"properties": everything_you_have_there}`.

The error is telling you that your schema is invalid because the description validator, which is a validator in JSON Schema, requires a string, and you gave it something else, a dict, the dict `{"type": "string}`. But you don't mean to use that validator at all, you're trying to define a property of your instance.

Hope that helps.

**Reply by jrmcmah2** [reply]
Thanks for the reply! I misread the my input body and did not see that the first level of key(s) only had the single key. I wanted to evaluate the dictionary that had the key "APPL" in my input. Thanks for the help pointing that out!
